test/e2e/test/elasticsearch/steps_mutation.go

```diff
@@ -193,8 +191,13 @@ func (hc *ContinuousHealthCheck) Start() {
 				return
 			case <-ticker.C:
 				ctx, cancel := context.WithTimeout(context.Background(), continuousHealthCheckTimeout)
+				client, err := hc.esClientFactory()
```

Comment:
`// recreate the Elasticsearch client at each iteration, since we may have switched protocol from http to https during the mutation`